src/flamenco/runtime/fd_executor.c

```diff
@@ -913,31 +913,68 @@ dump_instr_to_protobuf( fd_exec_txn_ctx_t *txn_ctx,
   } FD_SCRATCH_SCOPE_END;
 }
 
+/* https://github.com/anza-xyz/agave/blob/c4b42ab045860d7b13b3912eafb30e6d2f4e593f/sdk/src/transaction_context.rs#L319-L357 */
+int
```

Comment:
Can we also add a comment explaining what this function does and why we need it.

Reply:
Should explain the purpose of what these checks do + how we do it differently than agave (e.g. the lamport balancing)